subworkflows/local/bam_variant_calling_haplotypecaller/main.nf

```diff
@@ -126,31 +126,34 @@ workflow BAM_VARIANT_CALLING_HAPLOTYPECALLER {
 
         realigned_bam = BAM_MERGE_INDEX_SAMTOOLS.out.bam_bai
 
-        VCF_VARIANT_FILTERING_GATK(haplotypecaller_vcf.join(haplotypecaller_tbi),
-                    fasta,
-                    fasta_fai,
-                    dict,
-                    intervals_bed_combined,
-                    known_sites_indels.concat(known_sites_snps).flatten().unique().collect(),
-                    known_sites_indels_tbi.concat(known_sites_snps_tbi).flatten().unique().collect())
-
-        filtered_vcf = VCF_VARIANT_FILTERING_GATK.out.filtered_vcf.map{ meta, vcf-> [
-                [
-                    patient:meta.patient,
-                    sample:meta.sample,
-                    status:meta.status,
-                    sex:meta.sex,
-                    id:meta.sample,
-                    num_intervals:meta.num_intervals,
-                    variantcaller:"haplotypecaller"
-                ],
-                vcf
-            ]
+        if (!(skip_tools && skip_tools.split(',').contains('haplotypecaller_filter'))) {
+
+            VCF_VARIANT_FILTERING_GATK(haplotypecaller_vcf.join(haplotypecaller_tbi),
```

Comment:
should this also skip filtering for joint_germline?

Reply:
fair enough, can we name it more like haplotypecaller_skip_singlefilter ?  or so, I am expecting confusion when mixing `--joint_germline` and `skip_filter`

Reply:
we can skip filtering in `joint_germline` as well with this `--skip_tools haplotypecaller_filter`, it does make sense to do so

Reply:
actually not sure what to skip or not to skip there, prefer to do so in a separate PR

Reply:
I was thinking of the VQSR steps, but good for me to do so in a separate PR